src/libraries/System.Net.Http/src/System/Net/Http/HttpResponseMessage.cs

```diff
@@ -207,17 +207,8 @@ public override string ToString()
             return sb.ToString();
         }
 
-        private static bool ContainsNewLineCharacter(string value)
-        {
-            foreach (char character in value)
-            {
-                if ((character == HttpRuleParser.CR) || (character == HttpRuleParser.LF))
-                {
-                    return true;
-                }
-            }
-            return false;
-        }
+        private static bool ContainsNewLineCharacter(string value) =>
```

Comment:
We should be able to switch the one caller here to use [`HttpRuleParser.ContainsNewLine`](https://github.com/dotnet/runtime/blob/6a8f16e54fe2f91ef6457c94cab830f3c31baf9d/src/libraries/System.Net.Http/src/System/Net/Http/HttpRuleParser.cs#L150) directly